resources/js/features/games/components/GameSetRequests/GameSetRequestsRoot.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FC } from 'react';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { GameBreadcrumbs } from '@/common/components/GameBreadcrumbs';
+import { GameHeading } from '@/common/components/GameHeading';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { PatreonSupporterItem } from '@/features/patreon-supporters/components/PatreonSupporterItem';
+
+export const GameSetRequestsRoot: FC = () => {
+  const { game, deferredRequestors, initialRequestors, totalCount } =
+    usePageProps<App.Community.Data.GameSetRequestsPageProps>();
+  const { t } = useTranslation();
+
+  // Combine initial and deferred requestors once deferred props finish loading.
+  const allRequestors = useMemo(() => {
+    const requestors = [...initialRequestors];
+    if (deferredRequestors) {
+      requestors.push(...deferredRequestors);
+    }
+
+    return requestors;
+  }, [initialRequestors, deferredRequestors]);
+
+  return (
+    <div className="flex flex-col gap-5">
+      <GameBreadcrumbs game={game} system={game.system} t_currentPageLabel={t('Set Requests')} />
+      <GameHeading game={game} wrapperClassName="!mb-1">
+        {t('Set Requests')}
+      </GameHeading>
+
+      <div>{'A set for this game has been requested by the following users:'}</div>
+
+      <div>
+        <h2 className="border-b-0 text-center text-sm font-semibold sm:text-left sm:text-xl">
+          {t('All requestors ({{val, number}})', { val: totalCount })}
+        </h2>
```

Comment:
I'm not sure we want an "All requestors" subheading if there are no other subheadings. Perhaps the count can be merged into the "...by the following users:" string.

Conversely, maybe it would be interesting to have a "Lastest requestors" list, but I don't think that would provide any real benefit - and would be pointless of most games that only have a handful of requestors (or less).